src/Account.js

```diff
@@ -53,14 +53,17 @@ class Account {
             password_digest
         ) == ${JSON.stringify(password)} ?
         ${this.model}.find_by(JSON.parse('${JSON.stringify(attributes)}')).uuid :
-        {}
+        nil
```

Comment:
This was causing the app to seize up with an error when someone entered:
* a valid email
* an invalid password.